packages/@okta/vuepress-site/docs/guides/password-import-inline-hook/main/index.md

```diff
@@ -10,14 +10,14 @@ This guide provides a working example of an Okta Password Import Hook. It uses t
 
 **Learning outcomes**
 
-* Understand the Okta Password Import Inline Hook calls and responses
-* Implement a simple working example of a Password Import Inline Hook with a Glitch.com project, which acts as an external service
-* Preview and test a Password Import Inline Hook
+* Understand the Okta Password Import Inline Hook calls and responses.
+* Implement a simple working example of a Password Import Inline Hook with a Glitch.com project.
+* Preview and test a Password Import Inline Hook.
 
 **What you need**
 
-* A [Glitch.com](https://glitch.com) project or account.
-* An Okta developer org. [Create an org for free](https://developer.okta.com/signup/).
+* [Glitch.com](https://glitch.com) project or account
```

Comment:
Probably a very minor observation, but in some "Learning outcomes" sections for hooks, you start with the Okta org. In others, you start with "glitch" or "ngrok". Maybe for the sake of consistency, we could always start with the Okta org?

Reply:
Good eye! I'll start with the dev org first. Thanks.